pkgs/development/python-modules/frameioclient/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, requests
+, urllib3
+, importlib-metadata
+}:
+
+buildPythonPackage rec {
+  pname = "frameioclient";
+  version = "0.7.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "3ec2dd0be305a6249f365b23210ec8f554288752899bceae82c133264c720092";
+  };
+
+  propagatedBuildInputs = [
+    requests
+    urllib3 ] ++ lib.optionals (pythonOlder "3.8") [
```

sorry last one
```suggestion
    urllib3
  ] ++ lib.optionals (pythonOlder "3.8") [
```
you can do the change locally, and do:
```
git add pkgs/development/python-modules/frameioclient/default.nix
git commit --amend --no-edit
git push .. .. --force
```
and that should keep your githistory tidy

---

LGTM otherwise